apps/server/src/stores/runtimeState.ts

```diff
@@ -183,7 +181,7 @@ export function load(
   }
 
   // load events in memory along with their data
-  loadNow(timedEvents, eventIndex);
+  loadNow(rundown, timedEvents, eventIndex, prevBlock);
```

Comment:
Instead of passing conditional data. How about creating a loadCurrentBlock function to encapsulate the logic?